packages/amplify-category-function/src/provider-utils/awscloudformation/utils/permissionMapUtils.ts

```diff
@@ -0,0 +1,13 @@
+import _ from "lodash"
+
+export const fetchPermissionCategories = (permissionMap) => {
+  return _.keys(permissionMap);
+}
+
+export const fetchPermissionResourcesForCategory = (permissionMap, category: string) => {
+  return _.keys(_.get(permissionMap, [category]));
+}
+
+export const fetchPermissionsForResourceInCategory = (permissionMap, category: string, resourceName: string) => {
+  return _.get(permissionMap, [category, resourceName])
+}
```

Comment:
nit: `;` after line 12, newline at end of file

Reply:
will fix that and push